styles/styles.css

```diff
@@ -0,0 +1,336 @@
+body {
+  margin: 0;
+  width: 100%;
+}
+
+/*
+header {
+  background-color: #0A3C61;
+  height: 21rem;
+  text-align: left;
+  display: flex;
+  justify-content: center;
+}
+*/
+header {
+  background-color: 10, 11, 12 ;
+  height: 24rem;
+  justify-content: flex-start;
+
+}
+
+header h1 {
+  color: black;
+  position: absolute;
+  left: 13rem;
+  top: 1rem;
+  font-size: 60px;
+  font-family: sans-serif;
+}
+
+header h2 {
+  color: white;
+  position: absolute;
+  font-size: 25px;
+  font-family: sans-serif;
+  font-style: italic;
+  line-height: 36px;
+}
+.header-images {
+  width: 100%;
+  height: 24rem;
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+
+.b-box {
+  width: 30rem;
+  height: 100px;
+}
+
+
+.header-img {
+  object-fit: cover;
+  width: 100%;
+  height: inherit;
+  object-position: top;
+}
+
+
+
+.ftrlogo {
+  height: 30%;
+}
+
+.background2Box {
```

Comment:
Make sure to use semantic names for classes. It's hard to know what this is at a glance